xla/stream_executor/dnn.cc

```diff
@@ -250,6 +251,45 @@ DnnSupport::FusedMHAScaleBiasSoftmaxRunnerFromDesc(
       "FusedMHAScaleBiasSoftmaxRunnerFromDesc not implemented.");
 }
 
+tsl::StatusOr<std::unique_ptr<const dnn::FusedMHASoftmaxBackwardRunner>>
+DnnSupport::FusedMHASoftmaxBackwardRunnerFromDesc(
+    Stream* stream, const dnn::AlgorithmDesc& algorithm_desc,
+    dnn::FusedMHAKind kind,
+    const MatmulTensorDescriptor& bmm1_grad_gemm1_rhs_descriptor,
+    const MatmulTensorDescriptor& bmm1_grad_gemm2_rhs_descriptor,
+    const MatmulTensorDescriptor& bmm2_grad_gemm1_lhs_descriptor,
+    const MatmulTensorDescriptor& bmm2_grad_gemm2_rhs_descriptor,
+    const MatmulTensorDescriptor& d_output_descriptor,
+    const TensorDescriptor& d_bmm1_lhs_descriptor,
+    const TensorDescriptor& d_bmm1_rhs_descriptor,
+    const TensorDescriptor& d_bmm2_rhs_descriptor,
+    const TensorDescriptor& d_S_descriptor,
+    std::optional<dnn::TensorDescriptor> d_bias_descriptor, double scale,
+    std::optional<double> dropout_rate, std::optional<int64_t> seed) {
+  return tsl::errors::Unimplemented(
```

Comment:
For all the `tsl::errors::*` could you use the `absl` equivalent? like `absl::UnimplementedError` in this case

Reply:
I pushed a new commit to address this, but somehow this pr doesnt reflect that commit..

Reply:
I noticed this myself, I got an email when you pushed the commit but couldn't see the changes. Not sure what's going on!

Reply:
seems like it's still not showing up. I will do a rebase and see if that helps

Reply:
there we go

Reply:
Looks like that worked, nice!